scripts/common/host-packages.sh

```diff
@@ -95,6 +95,22 @@ function dpkg_install_host_packages() {
     _dpkg_install_host_packages "$dir" "$dir_prefix" "${packages[@]}"
 }
 
+function _dpkg_apt_get_status_and_maybe_fix_broken_pkgs() {
+    logStep "Checking package status"
+    if  sudo apt-get check status ; then
+        logSuccess "Status checked successfully. No broken packages were found."
+        return
+    fi
+
+    logWarn "Attempting to correct broken packages by running sudo apt-get install --fix-broken --yes"
+    sudo apt-get install --fix-broken --yes
```

Comment:
Remove sudo

Reply:
This has the potential to not work on an airgapped instance. Let's wait for the host packages proposal before addressing this.

```
$ sudo apt-get install --fix-broken --yes
Reading package lists... Done
Building dependency tree
Reading state information... Done
Correcting dependencies... Done
The following packages were automatically installed and are no longer required:
  libatasmart4 libblockdev-fs2 libblockdev-loop2 libblockdev-part-err2 libblockdev-part2
  libblockdev-swap2 libblockdev-utils2 libblockdev2 libmbim-glib4 libmbim-proxy libmm-glib0
  libnspr4 libnss3 libnuma1 libparted-fs-resize0 libqmi-glib5 libqmi-proxy libudisks2-0 libxmlb2
  usb-modeswitch usb-modeswitch-data
Use 'sudo apt autoremove' to remove them.
The following additional packages will be installed:
  libperl5.30 perl perl-modules-5.30
Suggested packages:
  perl-doc libterm-readline-gnu-perl | libterm-readline-perl-perl make libb-debug-perl
  liblocale-codes-perl
The following packages will be upgraded:
  libperl5.30 perl perl-modules-5.30
3 upgraded, 0 newly installed, 0 to remove and 4 not upgraded.
Need to get 6914 kB of archives.
After this operation, 12.3 kB of additional disk space will be used.
0% [Connecting to us-central1.gce.archive.ubuntu.com (35.184.213.5)]
```

Reply:
HI @emosbaugh,

Today it will fail ALWAYS when a conflict be faced or worst finish the install and does not let the user know.
The changes here allows the package manager remove a host package if/when required and try to sorted out when possible. It does not matter if is airgap or not. If the package manager has the dependency it will be sorted out. Otherwise, it would fail at the same away.

Reply:
you will need to remove sudo. the script will always be invoked as root or with sudo.